mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/social/ChessMatch.kt

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.sdp.mobile.ui.features.social
+
+class ChessMatch(
+    val adv: String,
+    val matchResult: MatchResult,
+    val cause: MatchResult.Reason,
+    val numberOfMoves: Int
+)
+
+enum class MatchResult {
+  WIN,
+  LOSS,
+  TIE;
+  enum class Reason {
+    CHECKMATE,
+    FORFEIT
+  }
+}

```

Comment:
I think `MatchResult.TIE` will not have any `Reason`. Maybe a `sealed class` would better represent the kind of results we can expect from a `ChessMatch`.